packages/plugins/rubygem-llhttp-ffi/rubygem-llhttp-ffi.spec

```diff
@@ -0,0 +1,92 @@
+# template: default
+#
+# PLEASE UPDATE WITH CARE!
+# The directory structure is very special and
+# does not really align with any other package
```

Comment:
I don't see what's so different. This looks fairly standard to me for a gem with native extensions.